spaceallocator/spaceallocator.go

```diff
@@ -175,11 +188,11 @@ func (allocator *Allocator) AllocateSpace(size uint64) (*Space, error) {
 		return nil, err
 	}
 
-	return &Space{size: size, path: allocator.path, allocator: allocator}, nil
+	return &spaceInstance{size: size, path: allocator.path, allocator: allocator}, nil
 }
 
 // Accept accepts previously allocated space.
-func (space *Space) Accept() error {
+func (space *spaceInstance) Accept() (err error) {
```

Comment:
why you chaged to named return in this case?

Reply:
Redundant changes. Initially, there were intermediate commits that I removed at final PR. Fixed.